authServer.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const dotenv = require('dotenv');
```

Comment:
I used dotenv because didn't know that was a build-in solution. However, I can't make work the --env solution. I think is a problem node version, I'm currently using 18.13.0.